### ISSUE TITLE
PEP 541 Request: bidule

### ISSUE THREAD

**Issue by parmentelat** [pypi/support#5297]
### Project to be claimed

`bidule`: https://pypi.org/project/bidule

### Your PyPI username

`parmentelat`: https://pypi.org/user/parmentelat

### Reasons for the request

Hello
I am writing to reclaim the name `bidule`
As you can see this is a stillborn project with a single release numbered 0.0.0 and with no description
I am the author of [a MOOC on Python (in French on France Université Numérique)](https://www.fun-mooc.fr/en/cours/python-3-des-fondamentaux-aux-concepts-avances-du-langage/)
and I have been using this name in a lesson on .. Python packaging
I am guessing the current state of affairs is the result of one of my students running the scenario till the end :)

---

I have just updated this lesson to show how to leverage `pyproject.toml` instead of the previous version that was still using `setup.py`
(this MOOC is quite old now, but is still maintained); to this end I have rebuilt [a dummy project on github here](https://github.com/flotpython/bidule), 
and was intending to publish it on PyPI as a live demo

---

I would have liked to reach the owner directly, but have not found a way to do that
So I was curious to know if you guys could help in the circumstance
Thanks in advance 

### Maintenance or replacement?

Replacement

### Source code repositories URLs

I do not know the current url, my guess there is none
my replacement sits here
https://github.com/parmentelat/bidule

### Contact and additional research

unfortunately for privacy reasons, as an author I cannot reach the students by name - actually I do not know any name, except when people talk on the community forum, which this person apparently has not done
so bottom line is, so far I have been unable to reach them

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @parmentelat,

We are working to contact the owner to determine if they're reachable and have sent them a courtesy message.

Final determination regarding project ownership will be made by the PyPI Administrators when the research process is complete.

**Comment by Thespi-Brain** [comment]
**bidule** has been deemed invalid. Additionally, the owner did not respond back regarding the status of their project. 

I recommend that the PyPI Admins release the name for future use.

**Comment by parmentelat** [comment]
Hello there

thanks for the update
apparently as of now the name is still booked on pypi, is there anything I need to do to be able to grab it ?

or what am I missing ?

thanks in advance !

**Comment by Thespi-Brain** [comment]
Hi @parmentelat,

It is now pending action from PyPI admins for the transfer process. You don't need to do anything. Thanks for your patience!

**Comment by ewdurbin** [comment]
Hello @parmentelat, I notice that the repository you intend to replace with is an equivalently "empty" project with no functionality. By releasing the name `bidule` to you to publish a similarly empty project, it would be open to transfer under PEP 541 to another party.

Do you have an intended project to upload to this project name which has functionality? If not, for demonstration purposes test.pypi.org is encouraged.

**Comment by parmentelat** [comment]
Hey @ewdurbin 

my repo is extremely lightweight all right, but I'm not sure I would call it an empty project
it is a demonstration project indeed, so there's not much in there, but it is fully functional and comes with a runnable main, a class, helper functions and a version number

it is a companion material for a section in a MOOC that runs on fun-mooc.fr here
https://www.fun-mooc.fr/en/cours/python-3-des-fondamentaux-aux-concepts-avances-du-langage/
and its primary focus is to show how to use `pyproject.toml` 
particularly in development mode, and to ultimately publish on pypi

in this respect it is not at crucial that it actually sits on pypi; so much so that I had not, at the time, deemed it necessary
however I had not anticipated that a student would go as far as grabbing that name (otoh that's silly of me because out of the 100_000+ students who have registered, it was almost certain that one of them would eventually do so;)

anyways; up to you; if you free the name I will use it with my stuff, which admittedly is not much, but way less empty than the current material :) if you don't, the name will remain taken by a totally ghost material; what's the best option ?

ps1. using test.pypi is an interesting suggestion but I am afraid it would blur the message by adding unneeded complexity
ps2. I might add tests in the repo to make it a little less hollow, but that won't affect the pypi content, obviously

anyways, thanks for your time :)

**Comment by Thespi-Brain** [comment]
Hi @parmentelat,

Following up about whether you were able to update your public repo of your project? As the admins mentioned, before we can transfer the project to you, we need proof of an active project. I will keep this issue open for one week unless I hear from you earlier.

**Comment by parmentelat** [comment]
Hi @Thespi-Brain 

I'm not sure I got that right, but I just pushed a few commits onto the github repo, if that's what was required ?

Let me know if anything else is needed on my end

Thanks !

**Comment by Thespi-Brain** [comment]
Hi @parmentelat,

We need more than just a few commits, because we don't want to run the risk of having another "empty" project. Hope this makes sense, thanks.

**Comment by Thespi-Brain** [comment]
Closing due to non response.
